src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -522,34 +522,26 @@
 	background-color: var(--vscode-terminal-hoverHighlightBackground);
 }
 
-.monaco-workbench .editor-instance .force-scrollbar .xterm .xterm-viewport,
-.monaco-workbench .pane-body.integrated-terminal .force-scrollbar .xterm .xterm-viewport,
-.monaco-workbench .editor-instance .xterm.focus .xterm-viewport,
-.monaco-workbench .pane-body.integrated-terminal .xterm.focus .xterm-viewport,
-.monaco-workbench .editor-instance .xterm:focus .xterm-viewport,
-.monaco-workbench .pane-body.integrated-terminal .xterm:focus .xterm-viewport,
-.monaco-workbench .editor-instance .xterm:hover .xterm-viewport,
-.monaco-workbench .pane-body.integrated-terminal .xterm:hover .xterm-viewport {
+.force-scrollbar .xterm .xterm-viewport,
+.xterm.focus .xterm-viewport,
+.xterm:focus .xterm-viewport,
+.xterm:hover .xterm-viewport {
```

Comment:
Let's bring back `.monaco-workbench` everywhere

Reply:
Oop, missed that